py/desitarget/cuts.py

```diff
@@ -195,6 +195,8 @@ def isGAIA_STD(ra=None, dec=None, galb=None, gaiaaen=None, pmra=None, pmdec=None
     nside : :class:`int`, optional, defaults to 2
         (NESTED) HEALPix nside, if targets are being parallelized.
         The default of 2 should be benign for serial processing.
+    dr : :class:`str`, optional, defaults to "dr2"
```

Comment:
Consider reverting all changes in `isGAIA_STD()`.

Reply:
Reverted.